File: python/raindrops/raindrops.py
def convert(number):
    numbers = {
        3: 'Pling',
        5: 'Plang',
        7: 'Plong'
    }

    raindrop = ''
    for key in numbers:
        if number % key == 0:
            raindrop += numbers[key]

    if raindrop == '':
        raindrop = number

    return str(raindrop)

File: solutions/php/raindrops/1/raindrops.php
<?php

function raindrops(int $number): string {
    $numbers = [
        3 => 'Pling',
        5 => 'Plang',
        7 => 'Plong',
    ];

    $raindrop = '';
    foreach ($numbers as $key => $item) {
        if ($number % $key === 0) {
            $raindrop .= $numbers[$key];
        }
    }

    return $raindrop ?: $number;
}